### ISSUE TITLE
Make wordninja caps aware?

### ISSUE THREAD

**Issue by paulperry** [keredson/wordninja#14]
Is there a way to make wordninja aware of capital letters and be able to cut at caps boundaries if not informed by the dictionary? Or is there a way to build a caps aware dictionary?

for example: `rCBVmeanSD`  should be cut as `rCBV` `mean` `SD`. But of course my domain specific language model must both define `mean` and `means` and the resulting cut is `rCBV`', '`meanS`', '`D`'.

Thoughts?

**Comment by keredson** [comment]
Yeah I'm pretty sure the existing model and code is case blind.  I'd be
open to a PR and a language model trained on case.  The statistical model
should be equivalent just the language would be larger.
